Update README for dev container start paths (#7)
reboot-dev/resemble-hello-world: README.md

subdirectories by proto package, while backend specific code can be
found in `backend/` and web specific code in `web/`.

> This repository also contains a [Dev Container](https://containers.dev/) configuration that will install Resemble
> requirements automatically. This configuration is designed to be run with
> VSCode, which has built-in Dev Container support.

Reviewer: ```suggestion
This repository includes a [Dev Container](https://containers.dev/) that _has all of the dependencies you need to build and run code in this repository already installed_.
```

> [!NOTE]
> The Dev Container's configuration for this repository is found in
> [`.devcontainer/devcontainer.json`](main/.devcontainer/devcontainer.json). You
> may expand on it to customize your development environment to your
> liking.

You can start the Dev Container in two different ways.

> <a id="use-a-github-codespace"></a>
> ## Use a GitHub Codespace with Local VSCode

Reviewer: ```suggestion
<a id="use-vscode-connected-to-a-github-codespace"></a>
## Use VSCode connected to a GitHub Codespace
```

GitHub's [Codespaces](https://github.com/features/codespaces) are machines that

> are hosted in the cloud for you. They are based on a Dev Container that includes
> all of the dependencies you need to build and run code in this repository.

Reviewer: This is now spelled out earlier.
```suggestion
are hosted in the cloud for you. 
```

> [!IMPORTANT]

> > Codespaces for this repository must be run through your local VSCode, not in a browser window.

Reviewer: ```suggestion
> You must connect your local VSCode to the codespace, you can not use VSCode in a browser window.
```

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/reboot-dev/resemble-hello-world)
<br>
(Right-Click to open in new tab or window)

Now you're ready to [run the application](#run-the-application)!

> <a id="local-dev-container-with-vscode"></a>
> ## Start a local Dev Container with VSCode

Reviewer: ```suggestion
<a id="use-vscode-with-a-local-dev-container"></a>
## Use VSCode with a local Dev Container
```

> Currently, our Dev Container at
> [`.devcontainer/devcontainer.json`](.devcontainer/devcontainer.json)
> only works on x86 CPU architectures. <br>

Reviewer: ```suggestion
> [!IMPORTANT]
> Currently, our Dev Container at [`.devcontainer/devcontainer.json`](main/.devcontainer/devcontainer.json) **only works on x86 CPU architectures**.
```
